canal/sync.go

```diff
@@ -107,7 +105,6 @@ func (c *Canal) runSyncBinlog() error {
 				if e != ErrExcludedTable &&
 					e != schema.ErrTableNotExist &&
 					e != schema.ErrMissingTableMeta {
-					log.Errorf("handle rows event at (%s, %d) error %v", pos.Name, curPos, err)
```

Comment:
why not replace this with a call to `c.cfg.Logger.Errorf`?

Reply:
I can if you want!

Reply:
I don't think we should log errors in the library, just return them

Reply:
that seems like the consistent thing to do

Reply:
sure I will change